src/frontend/src/common/components/library/data_subview/components/CreateNSTreeModal/components/SampleIdInput/components/InputInstructions/index.tsx

```diff
@@ -7,29 +7,26 @@ const InputInstructions = (): JSX.Element => {
   return (
     <StyledWrapper>
       <CollapsibleInstructions
-        header="Add Samples by ID (optional)"
+        header="Force-Include Samples by ID (optional)"
         items={[
           <div key={0}>
+            <SemiBold>
+              Samples selected on the Sample table or added here will be
+              force-included on your tree without undergoing QC.
+            </SemiBold>{" "}
+            <NewTabLink href="https://docs.google.com/document/d/1qjS6gRaKwpiw8WE2LbBNh9-m5ZFXm9X_H7O6UsXXiAw/edit#heading=h.61og0yhc0ua3">
+              Learn More
+            </NewTabLink>
+          </div>,
+          <div key={1}>
             Add <SemiBold>GISAID IDs</SemiBold> (e.g. USA/CA-CZB-0000/2021 or
             hCoV-19/USA/CA-CZB-0000/2021),{" "}
             <SemiBold>CZ GEN EPI Public IDs</SemiBold>, or{" "}
             <SemiBold>CZ GEN EPI Private IDs</SemiBold> below to include samples
             in your tree.
           </div>,
-          <div key={1}>
-            IDs must be separated by tabs, commas, or new lines.
-          </div>,
           <div key={2}>
-            Depending on the Tree Type, add up to 2000 samples.{" "}
```

Comment:
Do we no longer have the limit? I see that it's gone from the copywriting in the ticket, but did the limit get removed?

Reply:
@Janeece  ^^

Reply:
@mmmmmmaya @vincent-czi - I synced with Dan, we didnt change anything with the limit (once they are in the dialog, we don't enforce the limit (it's more of a recommendation). 

After re-sycning with Dan on this, we want to add back in a bullet at the end of the list that says "Adding more than 2000 samples will increase the tree building run time."